cnxepub/adapters.py

```diff
@@ -351,7 +351,10 @@ def adapt_single_html(html):
     html_root = etree.fromstring(html)
 
     metadata = parse_metadata(html_root.xpath('//*[@data-type="metadata"]')[0])
-    id_ = metadata['cnx-archive-uri'] or 'book'
+    id_ = metadata['cnx-archive-uri']
+    if id_ is None:
+        raise ValueError('Metadata element did not contain a cnx-archive-uri', 
```

Comment:
Could we clarify the failure case here? Is this when a collection does not have a `cnx-archive-uri` metadata entry? How common is this?

Reply:
It occurred in the tests but I am not sure if it actually occurs in books. I can remove this if this does not arise when baking any of the books.